### ISSUE TITLE
IntersectionObserver looses events on chrome on windows

### ISSUE THREAD

**Issue by ktsaou** [netdata/netdata#3289]
Well,

I think PR #3280 should be made optional.

It seems that on windows chrome, Intersection Observer does not always report the visibility of all elements.

**Comment by ktsaou** [comment]
The charts that are considered visible by the intersection observer can be reviewed on browser console, with this:

```js
console.log(NETDATA.intersectionObserver.visible_targets);
```

I found 2 issues:

1. Since intersection observer is async, charts where rendered while the hidden - this made the chart libraries go crazy. It is fixed with #3290 which triggers a re-draw when this unhiding a chart that was rendered while hidden.

2. On 1 case, I found that the `visible_targets` array given above, had the wrong number of charts. I'll investigate a bit further before taking action.
